src/app/common/mailaddressinfo.spec.ts

```diff
@@ -0,0 +1,61 @@
+// --------- BEGIN RUNBOX LICENSE ---------
+// Copyright (C) 2016-2020 Runbox Solutions AS (runbox.com).
+// 
+// This file is part of Runbox 7.
+// 
+// Runbox 7 is free software: You can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the
+// Free Software Foundation, either version 3 of the License, or (at your
+// option) any later version.
+// 
+// Runbox 7 is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with Runbox 7. If not, see <https://www.gnu.org/licenses/>.
+// ---------- END RUNBOX LICENSE ----------
+
+import { MailAddressInfo } from '../common/mailaddressinfo';
+
+describe('MailAddressInfo', () => {
+    it('Construct with name and email', () => {
+        const ma = new MailAddressInfo('Test', 'test1@runbox.com');
+        // expect(ma.name).toBe('Test');
```

Comment:
Hm, shouldn't these be either uncommented or removed?

Reply:
Not sure what you mean, these are new tests..

Reply:
Right – shouldn't they be ran then? ;)

Reply:
Oh! those lines, yes.. true..